tests/integ/sagemaker/serve/test_schema_builder.py

```diff
@@ -99,3 +99,58 @@ def test_model_builder_negative_path(sagemaker_session):
         match="Error Message: Schema builder for text-to-image could not be found.",
     ):
         model_builder.build(sagemaker_session=sagemaker_session)
+
+
+@pytest.mark.skipif(
+    PYTHON_VERSION_IS_NOT_310,
+    reason="Testing Schema Builder Simplification feature",
+)
+def test_model_builder_happy_path_with_task_provided(sagemaker_session, gpu_instance_type):
+    model_builder = ModelBuilder(model="bert-base-uncased:fill-mask")
```

Comment:
Can you add one more model task in test? Like below

```
hf_models = [
        {
            "id": "bert-base-uncased",
            "task": "fill-mask"
        },
        {
            "id": "bert-large-uncased-whole-word-masking-finetuned-squad",
            "task": "question-answering"
        }
    ]

for hf_model in hf_models:
   ...
```

Reply:
Added in the latest commit.